packages/dx-react-scheduler/docs/guides/editing.md

```diff
@@ -61,9 +62,7 @@ You can also use the `onAddedAppointmentChange` event to initialize a new appoin
 
 ## Preprocess Changes
 
-To validate user input or prevent user actions, handle the `EditingState` plugin's `onCommitChanges` event.
-
-For example, you can show a confirmation dialog before an appointment is removed. To detect an attempt to remove an appointment, check the `deleted` parameter's value in the `commitChanges` handler:
```

Comment:
My mistake. This section is not good place to illustrate confirmation dialog.

For confirmation dialog, we should add a new section. For example `Validation Actions` or something else.

This section should be kept but we should add a new demo that illustrates `preprocess changes`. For example - Copy appointments by DnD with `Shift` key. Or something else when we use the `onCommitChanges` function in unusual cases.